pandas/tests/io/formats/style/test_to_latex.py

```diff
@@ -302,6 +302,34 @@ def test_multiindex_row_and_col(df_ext):
     assert result == expected
 
 
+@pytest.mark.parametrize(
+    "multicol_align, siunitx, exp",
+    [
+        ("naive-l", False, " & A & &"),
+        ("naive-r", False, " & & & A"),
+        ("naive-l", True, "{} & {A} & {} & {}"),
+        ("naive-r", True, "{} & {} & {} & {A}"),
+    ],
+)
+def test_multicol_naive(df, multicol_align, siunitx, exp):
+    ridx = MultiIndex.from_tuples([("A", "a"), ("A", "b"), ("A", "c")])
+    df = df.astype({"A": int})
```

Comment:
Is this conversion necessary?
It seems that ``A`` column is already of ``int`` type.

Reply:
correct seemed to be legacy code (from copy/paste)